src/main/kotlin/com/exactpro/th2/dataprocessor/zephyr/service/impl/scale/server/ZephyrScaleServerApiService.kt

```diff
@@ -74,6 +74,11 @@ class ZephyrScaleServerApiService(
         }.toCommonModel()
     }
 
+    private suspend fun getLastExecution(testRunKey: String) = client
+        .get<List<ObjectNode>>("$baseApiUrl/testrun/$testRunKey/testresults")
+        .maxByOrNull { it["id"].intValue() }
```

Comment:
Why don't use class instead of an object node?  It is more obvious to see what is expected from response when you have a defined class for it